chat-widget/src/components/chatbuttonstateful/ChatButtonStateful.tsx

```diff
@@ -61,8 +65,8 @@ export const ChatButtonStateful = (props: IChatButtonStatefulParams) => {
                 dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.OutOfOffice });
             }
         },
-        unreadMessageString: props.buttonProps?.controlProps?.unreadMessageString,
-        ...outOfOfficeButtonProps?.controlProps
+        unreadMessageString: outOfOfficeChatButtonProps?.controlProps?.unreadMessageString,
+        ...outOfOfficeChatButtonProps?.controlProps
```

Comment:
I recommend we reconsidered this pattern for passing values and computing in-fly,  Im more in favor to have a function getWhatever Value and compute inside the function, in that way I have a single point of modification and do not mess with the component.